src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private int threshold;
+    private int size;
+    private Node<K, V>[] table;
+
+    private static class Node<K, V> {
+        private int hash;
```

Comment:
Do we need this field?

Reply:
In the default implementation, the node constructor takes four parameters (hash, key, value, next pointer).

Reply:
![image](https://user-images.githubusercontent.com/79798035/114880155-7c778000-9e0a-11eb-90ac-2ec39198b316.png)